com.microsoft.mrtk.input/Assets/Prefabs/MRTK RightHand Controller.prefab

```diff
@@ -107,6 +115,46 @@ PrefabInstance:
       propertyPath: handNode
       value: 5
       objectReference: {fileID: 0}
+    - target: {fileID: 2572330550550829917, guid: a9ae130324a36134e99beedf50e7be6f, type: 3}
+      propertyPath: pinchPoseSource.hand
+      value: 2
+      objectReference: {fileID: 0}
+    - target: {fileID: 2940030942784507886, guid: a9ae130324a36134e99beedf50e7be6f, type: 3}
+      propertyPath: fallbackPalmPoseSource.hand
+      value: 2
+      objectReference: {fileID: 0}
+    - target: {fileID: 2940030942784507886, guid: a9ae130324a36134e99beedf50e7be6f, type: 3}
+      propertyPath: rayPoseSource.positionActionProperty.m_Reference
```

Comment:
Looks like some of these changes might need rebuilding with the name changes and reworks

Reply:
Should be fixed!